tests/helper_test.go

```diff
@@ -1,52 +0,0 @@
-package main
```

Comment:
💡 Please revert this to the location under one testSuite: i.e. `tests` folder will contain all the tests: 

The whole point of unit tests is to test the "external interface" (ie public functions) to your package. That is unit tests should always be "white box" tests (see White Box Testing). That way you can refactor your code and your tests won't break.

I think this is moved to the folder where all collector so that you have access to a private method which is not correct.